tests/integration/src/test/scala/mesosphere/marathon/integration/MesosAppIntegrationTest.scala

```diff
@@ -12,14 +12,30 @@ import mesosphere.marathon.raml.{App, Container, DockerContainer, EngineType}
 import mesosphere.marathon.state.PathId._
 import mesosphere.marathon.state.{HostVolume, VolumeMount}
 import mesosphere.mesos.Constraints.hostnameField
-import mesosphere.{AkkaIntegrationTest, WhenEnvSet, WaitTestSupport}
+import mesosphere.{AkkaIntegrationTest, WaitTestSupport, WhenEnvSet}
+import org.scalatest.concurrent.PatienceConfiguration.Interval
 import play.api.libs.json.JsObject
 
 import scala.collection.immutable.Seq
 import scala.concurrent.duration._
 
 class MesosAppIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonTest {
 
+  override lazy val mesosNumMasters = 1
+  override lazy val mesosNumSlaves = 2
+
+  override def afterAll(): Unit = {
+    // We need to start all the agents for the teardown to be able to kill all the (UNREACHABLE) executors/tasks
```

Comment:
I'd much rather move unreachable related tests to `UnrechableIntegrationTest`.

Reply:
I'm not so sure that we should do it. This test is more related to `wipe=true` flag rather than pod unreachable behaviour.

Reply:
Agree with @jeschkies here, `TaskUnreachableIntegrationTest` already has this agent on/off logic, and we should try to keep it at one test

Reply:
Well, semantically it belongs to pods-related tests.

Reply:
That is true but messing with the number of agents etc should be contained to one test IMHO. Also, the test don't really cover just the wipe but unreachable tasks. If we'd only test that a persistent pod which writes `hello\n` to a volume is wiped and restarted should not contain `hello\nhello\n` but `hello\n`.